Bio/bgzf.py

```diff
@@ -404,7 +404,7 @@ def BgzfBlocks(handle):
 
 
 def _load_bgzf_block(handle, text_mode=False):
-    """Internal function to load the next BGZF function (PRIVATE)."""
+    """Load the next BGZF function (INTERNAL PRIVATE)."""
```

Comment:
Apologies, I missed the word function was being misused here (my own mistake in the original text).

How about:

```python
"""Load the next BGZF block of compressed data (PRIVATE)."""
```